### ISSUE TITLE
do not style scrollbars by default

### ISSUE THREAD

**Issue by fiws** [marcosmoura/vue-material#46]
it might be a personal preference, but i don't like that vue-material styles the scrollbars by default.

2 reasons:
* they do not fade out (like they do on chrome mobile)
* They are hard to change back.

i guess the fix for this, would be to just comment out https://github.com/marcosmoura/vue-material/blob/master/src/core/stylesheets/core.scss#L3

**Comment by marcosmoura** [comment]
Thanks for reporting this.
You're right. Another reason: When using only a single component the scrollbars come for free with mdCore module. This is not good at all. This will change on 0.3.0. 
I have some ideas to deal with scrollbars:

#### A css class: 
``` html
<html class="md-scrollbar">
```

#### A css class and a configurable property in mdCore component: 
```javascript 
Vue.material.scrollBar = true 
```
In this scenario the md-scrollbar will be added to ```html``` tag by default.

#### As a component: 
``` html
<md-content></md-content>
``` 
In this case will be like a container ```div``` with some options:
* ```md-scroll``` 
* ```md-scroll-y```
*  ```md-scroll-x```

And this will work like on Angular Material.

What do you think?

**Comment by fiws** [comment]
:+1: sounds good for me! Do you plan to implement all 3 solutions or just one? I think the css class and the component are a good idea.
The config property seems unnecessary. (just my opinion again) But I'm fine with it, if it is false by default.

**Comment by marcosmoura** [comment]
I have plans to implement the first. 
The config is good for the theming engine, but I need to think about on the impacts that this will cause, like toggling this property. In this case the mdCore should remove/add the class if the property change? The md-content component is a must have, but I don't think that needs to be shipped with a md-scroll class by default.

But without doubt the scrollbar thing will be optional. :)

**Comment by marcosmoura** [comment]
Already implemented and will be shipped on the very next release (v0.3.0)
